pkg/parser/extended_syntax_parser.y

```diff
@@ -296,6 +313,12 @@ evaluate_clause
 | TO EVALUATE IDENT label_clause INTO IDENT { $$.ModelToEvaluate = $3; $$.EvaluateLabel = $4; $$.EvaluateInto = $6 }
 ;
 
+run_clause
+: TO RUN IDENT { $$.ImageName = $3; }
+| TO RUN IDENT CMD stringlist { $$.ImageName = $3; $$.Parameters = $5 }
+| TO RUN IDENT CMD stringlist INTO identlist { $$.ImageName = $3; $$.Parameters = $5; $$.OutputTables = $7 }
```

Comment:
Is there a justification of the fact that INTO should follow by `identlist` other than `stringlist`?

Reply:
If it's identlist, the SQL statement is:

```SQL
SELECT * FROM source_table ORDER BY creation_date
TO RUN a_data_scientist/ts_data_processor:1.0
CMD
  "process_with_two_output_tables",
  "--time_column=t",
  "--value_column=v",
  "--window_width=120"
INTO output_table_1, output_table_2;
```

If it's stringlist, the SQL statement is:

```SQL
SELECT * FROM source_table ORDER BY creation_date
TO RUN a_data_scientist/ts_data_processor:1.0
CMD
  "process_with_two_output_tables",
  "--time_column=t",
  "--value_column=v",
  "--window_width=120"
INTO "output_table_1", "output_table_2";
```

The first is cleaner and simpler.